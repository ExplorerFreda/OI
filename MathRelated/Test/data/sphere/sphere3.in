5
10809.000000 15518.500000 6696.000000 15491.500000 1613.000000
10857.000000 15612.000000 17410.000000 4433.000000 13741.000000
12242.000000 15635.500000 4585.000000 7394.000000 9509.500000
5163.500000 19187.500000 7684.500000 8652.000000 855.500000
3414.500000 3502.000000 8081.500000 13750.500000 16038.000000
4511.500000 14620.000000 10773.500000 30.000000 1658.000000
