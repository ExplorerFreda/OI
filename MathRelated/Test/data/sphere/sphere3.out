5698.802 9601.993 12783.126 9960.897 7097.059
